pysollib/games/gypsy.py

```diff
@@ -635,13 +635,9 @@ def dealCards(self, sound=False):
         if sound:
             self.game.startDealSample()
         n = 0
-        rows = self.game.s.rows
-        reserves = self.game.s.reserves
-        for i in range(len(rows)):
-            r1 = reserves[i]
-            r2 = rows[i]
-            if r1.cards:
-                r1.moveMove(1, r2)
+        for row, reserve in zip(self.game.s.rows, self.game.s.reserves):
+            if reserve.cards:
+                reserve.makeMove(1, row)
```

Comment:
The method you want here is moveMove, not makeMove.

Reply:
By bad. Thanks for spotting that!